lib/traits/detail_screen.dart

```diff
@@ -51,7 +54,7 @@ class DetailScreenWidget extends StatelessWidget {
     return Consumer<TraitDetailProvider>(
         builder: (_, traitBasedDeviceNotifier, child) {
       if (traitBasedDeviceNotifier.isLoading) {
-        return CircularProgressIndicator();
+        return Center(child: CircularProgressIndicator());
```

Comment:
~~Not sure why this change is needed, was there some visual change that makes this better?~~